content/ja/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,175 @@
+---
+title: CronJobを使用して自動化タスクを実行する
+min-kubernetes-server-version: v1.21
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+CronJobは、Kubernetes v1.21で一般利用(GA)に昇格しました。古いバージョンのKubernetesを使用している場合、正確な情報を参照できるように、使用しているバージョンのKubernetesのドキュメントを参照してください。古いKubernetesのバージョンでは、`batch/v1` CronJob APIはサポートされていません。
+
+{{< glossary_tooltip text="CronJob" term_id="cronjob" >}}を使用すると、{{< glossary_tooltip text="Job" term_id="job" >}}を時間ベースのスケジュールで実行できるようになります。この自動化されたJobは、LinuxまたはUNIXシステム上の[Cron](https://en.wikipedia.org/wiki/Cron)のように実行されます。
```

Comment:
```suggestion
{{< glossary_tooltip text="CronJob" term_id="cronjob" >}}を使用すると、{{< glossary_tooltip text="Job" term_id="job" >}}を時間ベースのスケジュールで実行できるようになります。この自動化されたJobは、LinuxまたはUNIXシステム上の[Cron](https://ja.wikipedia.org/wiki/Cron)のように実行されます。
```
?
https://ja.wikipedia.org/wiki/Cron is a redirect, but it's a redirect that works.